src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.php

```diff
@@ -82,25 +85,19 @@
             ->args([
                 abstract_arg('Default mode'),
             ])
-            ->tag('validator.constraint_validator', [
-                'alias' => EmailValidator::class,
-            ])
+            ->tag('validator.constraint_validator', ['alias' => 'validator.email'])
```

Comment:
Aliasing by FQCN is already the default so there is no need to duplicate. The intent waslikely to alias by service-id (as done for ExpressionValidator and a few others)

Reply:
This is *not* the expected thing. The Email constraint still return `EmailValidator::class` in its `validatedBy` method.

Reply:
Sure, but what is the alias for then? Note that `AddConstraintValidatorsPass` registers the validator under both keys:

```php
                $validators[$attributes[0]['alias']] = new Reference($id);
            }

            $validators[$definition->getClass()] = new Reference($id);
```

What's your advice? Remove the alias instead of adding one?